src/data/real/ennreal.lean

```diff
@@ -148,18 +154,47 @@ protected lemma zero_lt_one : 0 < (1 : ennreal) :=
 lemma two_ne_zero : (2:ennreal) ≠ 0 := (ne_of_lt zero_lt_two).symm
 lemma two_ne_top : (2:ennreal) ≠ ∞ := coe_two ▸ coe_ne_top
 
+/-- The set of `ennreal` numbers that are not equal to `∞` is equivalent to `ℝ≥0`. -/
+def ne_top_equiv_nnreal : {a | a ≠ ∞} ≃ ℝ≥0 :=
```

Comment:
What about mentioning this in the module docstring?

Reply:
... and creating a module doc string? 😄